routes/user.route.js

```diff
@@ -8,13 +8,13 @@ const router = Router();
 router.post('/login', noAuth, UserController.login);
 router.post('/register', noAuth, UserController.create);
 router.get('/logout', stdAuth, UserController.logout);
-router.post('/registerAdmin', adminAuth, UserController.createAdmin);
+router.post('/registerAdmin',  UserController.createAdmin);
```

Comment:
This is not secure, anyone could create an admin account for themselves, creating the first admin user should be done only in the database, then admins should be able to update existing users to give them admin access

Reply:
Nice observation. Should I wait to review once the changes have been made @TomStrong ??